### DISCUSSION TITLE
Application rejected 😖

### DISCUSSION THREAD

**Discussion by dantzlerd** [community/community#53177] [GitHub Education]
I'm an adjunct professor who teaches online courses, and my application(s) have been rejected. I have a verified edu e-mail account. I don't have a university issued ID, and my geolocation is not near the school. With my application, I once tried to upload an image of my paystub with redactions. I also attempted to complete another application with a screenshot of the learning environment, illustrating a recent date and my name and photo listed as the instructor of the course and that was also rejected. I do not receive much (if any) paper mail communication from the university, most communication is via e-mail. I am not sure what other supporting documentation I have that I can provide that would be acceptable. Is there any solution?

**Comment by samus-aran** [comment]
Hi @dantzlerd 👋 

I suspect the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈 

It would be preferable that a student/teacher is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students/teachers submit a live photo of their evidence rather than an upload. We understand that students/teachers may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Let us know here if you need further assistance
